src/components/Review/ReviewComment.tsx

```diff
@@ -8,22 +8,36 @@ import {
 import Loading from '../Loading'
 
 type ReviewCommentProps = {
-  reviewDecisionId: number
+  reviewDecisionId?: number
   isEditable: boolean
 }
 
 const ReviewComment: React.FC<ReviewCommentProps> = ({ reviewDecisionId, isEditable }) => {
   const [updateComment] = useUpdateReviewDecisionCommentMutation()
-  const { data, error } = useGetReviewDecisionCommentQuery({ variables: { reviewDecisionId } })
+  const { data, error } = useGetReviewDecisionCommentQuery({
+    variables: { reviewDecisionId: reviewDecisionId as number },
+    skip: !reviewDecisionId,
+  })
   const [comment, setComment] = useState('')
 
+  if (!reviewDecisionId) return null
+
   if (error) return <Message error title={strings.ERROR_GENERIC} list={[error]} />
 
   if (!data) return <Loading />
 
   const initialComment = data?.reviewDecision?.comment || ''
 
-  if (!isEditable) return <p>{initialComment}</p>
+  if (!isEditable)
```

Comment:
do we want to hide the comment ? (if it doesn't exist ?)

Reply:
Tried something. But haven't tested. In a rush : P

Reply:
Testing now

Reply:
Looks good to me, going to merge this one now